### PR TITLE
Update Tesseract repository (moved to Github)

### PR DESCRIPTION
**Pull request by francoisbruneau**
A couple of URL updates for those interested in trying out the HEAD version of Tesseract and its recognition data.

### CODE REVIEW

File: Library/Formula/tesseract.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Tesseract < Formula
   end
 
   head do
-    url 'https://code.google.com/p/tesseract-ocr/', :using => :git
+    url 'https://github.com/tesseract-ocr/tesseract/', :using => :git
```
Comment (DomT4) [inline Library/Formula/tesseract.rb]: Can just make this `url "https://github.com/tesseract-ocr/tesseract.git"` now.

File: Library/Formula/tesseract.rb [hunk 2]
```diff
@@ -128,6 +128,10 @@ class Tesseract < Formula
     sha1 "a559801a42be8a3dc15ae2ec27da99bd8b0b85ac"
   end
 
+  resource "tessdata-head" do
+    url 'https://github.com/tesseract-ocr/tessdata/', :using => :git
```
Comment (DomT4) [inline Library/Formula/tesseract.rb]: Likewise. `.git` here, not `:using => :git`.

File: Library/Formula/tesseract.rb (hunk 1 above)
Comment (francoisbruneau) [inline Library/Formula/tesseract.rb]: Done, thanks!

File: Library/Formula/tesseract.rb (hunk 2 above)
Comment (francoisbruneau) [inline Library/Formula/tesseract.rb]: Fixed!

File: Library/Formula/tesseract.rb [hunk 3]
```diff
@@ -128,6 +128,10 @@ class Tesseract < Formula
     sha1 "a559801a42be8a3dc15ae2ec27da99bd8b0b85ac"
   end
 
+  resource "tessdata-head" do
```
Comment (DomT4) [inline Library/Formula/tesseract.rb]: Let's tuck this inside the `head do` block just to make it clear it only applies to the `head` logic. You should be able to drop the `resources.reject { |r| r.name == "tessdata-head" }.each` dance below as well then.

File: Library/Formula/tesseract.rb [hunk 4]
```diff
@@ -128,6 +128,10 @@ class Tesseract < Formula
...
+    url 'https://github.com/tesseract-ocr/tessdata.git'
```
Comment (DomT4) [inline Library/Formula/tesseract.rb]: And double quotes around here. Everything should be wrapped in double quotes, we've been moving across more strictly to the Ruby style guide, but new lines in particular shouldn't reinforce the single quotes :)

**Pull request closed without merging**